src/app/src/main/kotlin/org/imperial/mrc/hint/db/SessionRepository.kt

```diff
@@ -76,6 +76,14 @@ class JooqSessionRepository(private val dsl: DSLContext) : SessionRepository {
 
     }
 
+    override fun removeSessionFile(sessionId: String, type: FileType, hash: String) {
+        dsl.deleteFrom(SESSION_FILE)
```

Comment:
This won't throw an error if no rows found? We should probably return a 404 if the hash doesn't exist for the session and file type?

Reply:
This was deliberate. I think it's better if idempotent. If it already doesn't exist, great! Means e.g. the front-end can call a "delete all" method without worrying about whether some exist and others don't.